index.js

```diff
@@ -1,6 +1,7 @@
 function stringify (obj, options) {
   options = options || {}
-  var indent = JSON.stringify([1], null, get(options, 'indent', 2)).slice(2, -3)
+  var JSONstringify = get(options, 'stringify', JSON.stringify)
+  var indent = JSONstringify([1], null, get(options, 'indent', 2)).slice(2, -3)
```

Comment:
This is a pretty hacky way to compute the indent, I know. It works for `JSON.stringify`, but I wonder how many other JSON stringify implementations it will work with. What do you think?

Reply:
Good point.I think we should use `JSON.stringify` here.

Reply:
Fixed